src/main/target/CC3D/target.h

```diff
@@ -72,8 +72,13 @@
 #define USE_VCP
 #define USE_UART1
 #define USE_UART3
+
+#ifdef CC3D_OPBL
```

Comment:
Yes, ok? 10 sections, is that a problem? Each section dealing with its item. Keeping config items together instead of scattering them all over the place. The way it us done now. Horrible.
Look how it ended up in doubles etc etc....